spec/uo_cli_spec.rb

```diff
@@ -149,13 +194,13 @@ def delete_directory_or_file(dir_or_file)
     end
 
     it 'post-processes a scenario' do
-      filename = File.join(test_directory, 'run', 'two_building_scenario', 'default_scenario_report.csv')
-      db_filename = File.join(test_directory, 'run', 'two_building_scenario', 'default_scenario_report.db')
+      filename = File.join(test_directory, 'run', 'one_building_scenario', 'default_scenario_report.csv')
```

Comment:
shouldn't we have at least 2 buildings in the scenario to ensure the aggregations are working?

Reply:
Yeah, it would make more sense for feature visualization too to have two features. It was reduced to 1 only because running the rake tasks was taking too long, since it was running 2 features for 3 geometry types